config_pgcluster.yml

```diff
@@ -179,7 +179,8 @@
               + [{ 'option': 'shared_preload_libraries', 'value': new_value }] }}
           vars:
             # Find the last item in postgresql_parameters where the option is 'shared_preload_libraries'
-            shared_preload_libraries_item: "{{ postgresql_parameters | selectattr('option', 'equalto', 'shared_preload_libraries') | list | last | default({ 'value': '' }) }}"
+            shared_preload_libraries_item: "{{ postgresql_parameters | selectattr('option', 'equalto', 'shared_preload_libraries') | list | last | default({ 'value':
+              '' }) }}"
```

Comment:
I don't understand the need for this change because a lot of spaces have been added here

I would not recommend changing such complex conditions